bigbluebutton/greenlight
# Meeting Starter now updates last session

## Description

`room.last_session` now gets updated with bbb response `createDate` when a Meeting is started. 
Somewhat in conjunction with the room status, we can now display the date of the last session.

## Testing Steps


1. Create a few Rooms
2. Go to Admin/Server Rooms
3. Start/End some Meetings. 
4. In Admin/Server Rooms, the list will update as the Rooms toggled between active and not running. 

- If a Meeting has never been started in the Room, it should display "No meeting yet"
- If a Meeting is currently active, it should display **Started: datetime**
- If a Room had a Meeting, but it is currently not running, it should display **Ended: datetime**



## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/43917914/178782567-e316920f-ca2a-49e0-8ca8-ca2418f72f21.png)
